### DISCUSSION TITLE
Can't access field containing a period character

### DISCUSSION THREAD

**Discussion by CoconutMacaroon** [grafana/grafana-json-datasource#187] [Q&A]
Trying to use the Alpha Vantage API. Here is an example JSON from the API (URL is `https://www.alphavantage.co/query?function=GLOBAL_QUOTE&symbol=SPY&apikey=[REDACTED]`
```json
{
    "Global Quote": {
        "01. symbol": "SPY",
        "02. open": "447.3200",
        "03. high": "448.3600",
        "04. low": "444.0200",
        "05. price": "447.1700",
        "06. volume": "74229850",
        "07. latest trading day": "2021-09-16",
        "08. previous close": "447.8800",
        "09. change": "-0.7100",
        "10. change percent": "-0.1585%"
    }
}
```
I'm trying to access field `10. change percent` with the query `$.Global Quote.10. change percent`, but it just says `No data`. How can I fix this query?

**Comment by marcusolsson** [comment]
I don't know, honestly. JSONPath doesn't seem to support quoting the properties. I made it work with JSONata, however. Would that work for you?

<img width="969" alt="Screenshot 2021-10-13 at 15 13 33@2x" src="https://user-images.githubusercontent.com/8396880/137139655-cf717bcb-d234-4bfd-a517-a98d2017bc76.png">
